app/_hub/kong-inc/mtls-auth/overview/_adding-cert-authorities.md

```diff
@@ -0,0 +1,47 @@
+---
+nav_title: Adding Certificate Authorities
```

Comment:
I'd move both of these guides into `/how-to/` instead of `/overview/`. The "Overview" section generally contains conceptual (and sometimes reference) topics.